### DISCUSSION TITLE
Rule issue

### DISCUSSION THREAD

**Discussion by uxhamby** [arendst/Tasmota#13716] [Q&A] [answered]
Should this rule not result in power toggling on & off once a minute?

11:30:18.361 MQT: stat/tasmota_019868/RESULT = {"Rule1":{"State":"ON","Once":"OFF","StopOnError":"OFF","Length":38,"Free":473,"Rules":"= on Time#Minute do power toggle endon"}}

I get no action at all. What am I missing here?

Program Version | 9.5.0(tasmota)
-- | --
2021-06-17T08:26:35
2_7_4_9/2.2.2-dev(38a443e)

Thanks.

**Comment by sfromis** [comment, marked as answer]
You are missing having correct rule syntax, it should not start with =
The syntax for the rule command does not include = to enter a rule set.

**Reply by uxhamby** [reply]
Thank you.
